python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -844,6 +844,11 @@ def test_union_classmethod_usage(self):
     def test_isinstance_dataframe(self):
         self.assertIsInstance(self.spark.range(1), DataFrame)
 
+    def test_checkpoint_dataframe(self):
```

Comment:
Shouldn't this be a bit more exhaustive?

Reply:
I enabled some more tests that were skipped